indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -872,7 +863,7 @@
    like this:
    {:indexs[{:name '' :settings {<shards and replicas>}}]
     :mapping{:properties{:example {:type 'keyword'}}}}
-   Note: Indexs normally has two items, the all revisions index and the normal index
+   Note: Indexs normally have two items, the all revisions index and the normal index
    Note: most mappings include a normal version and a lowercase version
```

Comment:
Indicies

Reply:
so my problem with this is that indicies does not show up in the code at all, but indexes, meaning I'm still wrong, but not in the way you want.

Reply:
indices is used 117 times
indexes is used 311 times
indexs is used 8 times
I will go with indexes